viewing_party/party.py

```diff
@@ -1,23 +1,166 @@
-# ------------- WAVE 1 --------------------
+from pprint import pprint
+#HELPER FUNCTION
+def _get_movies_watched_by_friends(user_data):
+    movies_watched_by_friends = []
+    
+    for friend in user_data["friends"]: #[{}]
+        for friend_movie in friend["watched"]: #{}
+            if friend_movie in movies_watched_by_friends:

```

Comment:
This conditional could be rewritten to avoid using `continue` in the `if` block, I believe as follows:
```python
            if friend_movie not in movies_watched_by_friends:
                movies_watched_by_friends.append(friend_movie)     
```